docs/docs/quick_start/advanced_features/disk_expansion.md

```diff
@@ -4,3 +4,88 @@ sidebar_label: "Disk Expansion"
 ---
 
 # Disk Expansion
+
+A storage system is ususally expected to expand its capacity by adding a new disk into a storage node. In HwameiStor, it can be done with the following steps.
```

Comment:
```suggestion
A storage system is usually expected to expand its capacity by adding a new disk into a storage node. In HwameiStor, it can be done with the following steps.
```